scrapy/utils/defer.py

```diff
@@ -2,6 +2,7 @@
 Helper functions for dealing with Twisted deferreds
 """
 import asyncio
+from functools import wraps
```

Comment:
:lipstick: Below `import inspect`?

Reply:
I like @Gallaecio's style proposal a bit more than what's in the code. Merging PR anyways, this is not a blocker.